Add Pulumi AI and Registry blog post (#2799)
pulumi/pulumi-hugo: themes/default/content/blog/2023-05-07-leveling-up-pulumi-ai-with-the-pulumi-registry/index.md

>
> Here's a Pulumi program in TypeScript that creates an AlloyDB Cluster with an automatic backup policy:
>
> ```typescript
> import * as pulumi from "@pulumi/pulumi";
> import * as gcp from "@pulumi/gcp";
>
> const projectId = gcp.config.project;
> const location = "us-central1";
>
> const network = new gcp.compute.Network("alloydb-network", {
>   autoCreateSubnetworks: false,
> });
>
> const alloydbCluster = new gcp.alloydb.Cluster("alloydbCluster", {
>   clusterId: "my-alloydb-cluster",
>   location,
>   project: projectId,
>   network: network.selfLink,
>   automatedBackupPolicy: {
>     enabled: true,
>     labels: {
>       "backup-type": "automated",
>     },
>     location: "us-central1",
>     quantityBasedRetention: {
>       count: 7,
>     },
>     timeBasedRetention: {
>       retentionPeriod: "7d",
>     },
>     weeklySchedule: {
>       daysOfWeeks: ["MONDAY", "WEDNESDAY", "FRIDAY"],
>       startTimes: [
>         {startTime: "04:00:00"},
>       ],
>     },
>   },
> });
>
> export const alloyDBClusterName = alloydbCluster.name;
> ```

## Getting Started with the Enhanced Pulumi AI

> Our enhancements to Pulumi AI have made it a more useful tool for generating accurate and relevant cloud infrastructure solutions. We're excited about the future of Pulumi AI as we continue to integrate it into [Pulumi Cloud](https://app.pulumi.com) and empower everyone to become a cloud engineer.

Reviewer: ```suggestion
Our enhancements to Pulumi AI have made it an even more useful tool for generating accurate and relevant cloud infrastructure solutions. We're excited about the future of Pulumi AI as we continue to integrate it into [Pulumi Cloud](https://app.pulumi.com) and empower everyone to become a cloud engineer.
```

> These enhancements are live now at [Pulumi AI](/ai/), and we've added links to Pulumi AI in pages on the Pulumi Registry, for example on the [`eks.Cluster`](/registry/packages/eks/api-docs/cluster/) docs mentioned earlier.

Reviewer: ```suggestion
These enhancements are live now at [Pulumi AI](/ai/).  We've also added "Explore with Pulumi AI" links to Pulumi Registry pages, for example on the [`eks.Cluster`](/registry/packages/eks/api-docs/cluster/) page mentioned earlier.
```

> Give them a try and share your experiences with us. Together, let's build a future where infrastructure as code becomes more accessible, efficient, and enjoyable for everyone.

Reviewer: Should we mention that we also added the ability to share your conversation results with others?  Perhaps include a screenshot like this:

<img width="496" alt="Screenshot 2023-05-17 at 9 53 28 PM" src="https://github.com/pulumi/pulumi-hugo/assets/223467/cfe5c070-1555-471e-b6c4-a66a6df0590c">

Author: Added
